apps/frontend/src/pages/people/[username].tsx

```diff
@@ -83,6 +87,17 @@ const CommentModal: React.FC<CommentModalProps> = ({
               {displayName}
             </span>
           </div>
+          <div className="flex flex-col mt-2">
+            <AppButton
+              onClick={async () => {
+                setHasTappedBack(true);
+                await onTapBack();
+              }}
+              disabled={hasTappedBack}
+            >
+              Share socials back
+            </AppButton>
```

Comment:
Hmm, I prefer that tapback is automatic so that the exchange is symmetric. It feels (1) like an equal relationship and (2) eliminates a step that might otherwise be skipped / forgotten. Just a personal preference, feel free to disregard.

Reply:
this is for consent, i.e. i dont want my data to ever be shared automatically. but in the future we should have this be a configurable option where someone can turn this toggle on in settings